lib/rules/rule-selector-property-disallowed-list/README.md

```diff
@@ -61,3 +61,27 @@ a { padding-top: 0px; }
 ```css
 html[data-foo] { color: red; }
 ```
+
+### `ignore: ["keyframe-selectors"]`
+
+Ignore keyframe selectors.
+
+Given:
+
+```json
+[{"/^[a-z]+$/": ["opacity"]}, { "ignore": ["keyframe-selectors"] ]
+```
+
+The following pattern is _not_ considered a problem:
+
+<!-- prettier-ignore -->
+```css
+@keyframes fade-in {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
```

Comment:
[suggest] Super-minor. We may save a few lines, although you can ignore this comment.

```suggestion
  from { opacity: 0; }
  to { opacity: 1; }
```